app/src/app/framework/models/cmdb-type-reference.ts

```diff
@@ -0,0 +1,31 @@
+/*
+* DATAGERRY - OpenSource Enterprise CMDB
```

Comment:
This is the old header. Make sure that you use the one with the current date from now on.